src/pytest/test_replicate.py

```diff
@@ -0,0 +1,73 @@
+import subprocess
+import signal
+import os
+import os.path
+from RLTest import Env
+import time
+
+class TimeLimit(object):
+    """
+    A context manager that fires a TimeExpired exception if it does not
+    return within the specified amount of time.
+    """
+    def __init__(self, timeout):
+        self.timeout = timeout
+    def __enter__(self):
+        signal.signal(signal.SIGALRM, self.handler)
+        signal.setitimer(signal.ITIMER_REAL, self.timeout, 0)
+    def __exit__(self, exc_type, exc_value, traceback):
+        signal.setitimer(signal.ITIMER_REAL, 0)
+        signal.signal(signal.SIGALRM, signal.SIG_DFL)
+    def handler(self, signum, frame):
+        raise Exception()   
+
+def testDelReplicate():
+  env = Env(useSlaves=True)
+  master = env.getConnection()
+  slave = env.getSlaveConnection() 
+
+  master.execute_command('set foo bar')
```

Comment:
this line should be removed - otherwise the existence of foo=bar does not give us any significance!